src/components/HorizontalBarChart/styles.css

```diff
@@ -34,10 +34,14 @@
   text-align: right;
 }
 
-.bar-chart-value-lost {
+.bar-chart-value.isLostRow {
   background-color: var(--color-red);
 }
 
+.bar-chart-value.isLongValue {
+  min-width: 95px;
```

Comment:
I think this could backfire on mobile, but for now I get that it's a good first approach.

Reply:
yes for now I recommend playing on big screens, I already have a card to fix everything for smaller devices later 😅 

<img width="339" alt="image" src="https://user-images.githubusercontent.com/8755889/211591330-d2302053-16d2-4fe5-9178-7002ae9cc6b4.png">